ote/src/cljs/ote/app/controller/transport_operator.cljs

```diff
@@ -386,16 +377,18 @@
         (assoc app :transport-operator-loaded? false))
       (assoc app :transport-operator-loaded? true)))
 
-  EditTransportOperatorResponse                             ;; todo: pois
+  EditTransportOperatorResponse
   (process-event [{response :response} app]
     (let [state (assoc app :transport-operator response
-                           :transport-operator-loaded? true)
+                           :transport-operator-loaded? true
+                           :ytj-response {})
           nap-business-id (get-in state [:transport-operator ::t-operator/business-id])
           op-ytj-cache-miss? (or (empty? (:ytj-company-names state)) (not= nap-business-id (get-in state [:ytj-response :businessId])))]
-
-      (if op-ytj-cache-miss?
-        (send-fetch-ytj state nap-business-id)
-        (process-ytj-data state (:ytj-response state)))))
+      (if (flags/enabled? :open-ytj-integration)
```

Comment:
Should we turn the flag off from testing so we could make cypress tests work in CI and test server?

Reply:
Yes